### ISSUE TITLE
Inssue on Cargo Build

### ISSUE THREAD

**Issue by n8acl** [jess-sys/dmr-bridge-discord#1]
Got Rust installed and am trying to install the DMR-Bridge-Discord on my server... I am getting the following error when I try to run cargo build --release

  Compiling audiopus_sys v0.1.8
   Compiling regex v1.5.4
error: failed to run custom build command for `audiopus_sys v0.1.8`

Caused by:
  process didn't exit successfully: `/root/dmr-bridge-discord/target/release/build/audiopus_sys-f6a2b9691d7bf68a/build-script-build` (exit status: 101)
  --- stdout
  cargo:info=No feature or environment variable found, linking by default.
  cargo:rerun-if-env-changed=OPUS_NO_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR
  cargo:info=`pkg_config` could not find `Opus`.
  cargo:info=Opus source path: "/root/.cargo/registry/src/github.com-1ecc6299db9ec823/audiopus_sys-0.1.8/opus".
  cargo:info=Opus will be built as dylib-library.
  Updating build configuration files, please wait....

  --- stderr
  autogen.sh: 14: autoreconf: not found
  thread 'main' panicked at 'Failed to autogen Opus.', /root/.cargo/registry/src/github.com-1ecc6299db9ec823/audiopus_sys-0.1.8/build.rs:102:9
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...

This is on ubuntu 22.04

**Comment by ThalusA** [comment]
Thanks for the feedback, the errors come from one of its packages but I can't manage to fix it so I made an issue inside their repository https://github.com/Lakelezz/audiopus_sys/issues/10. I will try to fix it nevertheless.

**Comment by ThalusA** [comment]
Ok nevermind the problems come from me not having installed `libopus` on Ubuntu you have to install `libopus-dev` with `apt`

**Comment by ThalusA** [comment]
https://github.com/jess-sys/dmr-bridge-discord#build README.md is now updated

**Comment by n8acl** [comment]
GOt ya... ok let me try that and I will let you know :) thanks :)

**Comment by n8acl** [comment]
Alright I got that installed and it compiled... Now I am getting this error.... 

thread 'main' panicked at 'Expected a target tx address in the environment: NotPresent', src/commands/transmitter.rs:39:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

I tried to download the precompiled release too (just in case) and still got the same error.... :)

**Comment by jess-sys** [comment]
Do you have a file named .env with the variables specified in the README ? This file should be placed in the same directory you’re running the executable.

This error should occur only when these variables are not set in the file.

**Comment by n8acl** [comment]
I did indeed... 

HEre is what the file looks like:

# Analog Bridge configuration
TARGET_RX_ADDR="10.0.0.4:32002"
LOCAL_RX_ADDR="127.0.0.1:34002"

# Discord Bot configuration
BOT_TOKEN="<My Bot Token>"
BOT_PREFIX="!"

Obviously I didn't incude my Bot Toke in this post.

**Comment by ThalusA** [comment]
https://github.com/jess-sys/dmr-bridge-discord/commit/bfd92fff6ffdb2af45fc69eae33b2cedc271ec1e Sorry it was my bad, there was a typo, pull the changes and retry it?

**Comment by n8acl** [comment]
There we go.. Built and was able to run it. Outstanding! Thanks so much for all the quick changes. I really appreciate it.

What I have been trying to do it, since I realized that this bot uses USRP to connect to Analog Bridge, which connects to MMDVM_bridge to bridget o DMR, I just changed where USRP points.....

Now I am able to bridge an AllstarLink Node to Discord... I just bypass Analobg Bridge and MMDVM_Bridge and just point it at the USRP channel in Allstar. So now this is awesome... and the audio sounds great!
